frontend/src/pages/Home/Home.tsx

```diff
@@ -1,24 +1,79 @@
-import { FloatButton, Layout, Typography } from 'antd'
+import { Button, FloatButton, Layout, List, Typography } from 'antd'
+import { useItems } from 'services/item_service'
 import React from 'react'
 import { Link, Outlet } from 'react-router-dom'
 
+import { useActivities, useUpdateActivity } from 'services/activities_service'
+import { usePagedResultData } from 'services/utils'
+
 export interface HomeProps {}
 
 const Home = ({}: HomeProps) => {
+
+    const { data: recentsPagedResult, isPending: recentsPending, fetchStatus: recentsStatus } = useItems(1, 5, undefined, undefined, {key: "created", order: "descend"})
+    const { data: recents} = usePagedResultData(recentsPagedResult)
+
+    const { data: upcomingPagedResult, isPending: upcomingPending, fetchStatus: upcomingStatus } = useActivities(1, 5, undefined, {completed:['false'], pending:['true']})
+    const { data: upcoming } = usePagedResultData(upcomingPagedResult)
+
+    const { data: unfinishedPagedResult, isPending: unfinishedPending, fetchStatus: unfinishedStatus } = useActivities(1, 5, undefined, {completed:['false'], pending:['false'], hasEndTime:['false']})
+    const { data: unfinished } = usePagedResultData(unfinishedPagedResult)
+
+    const activityUpdateMutation = useUpdateActivity()
+
+    const markFinished = (item:any) => {
+        activityUpdateMutation.mutate({
+            token: item.token,
+            patch: { pending: false, finished: true}
+        })
+    }
+
     return (
         <div style={{}}>
             <Typography.Title level={2}>Recently Added</Typography.Title>
-            {[1, 2, 3].map(x => (
-                <div key={x}>{x}</div>
-            ))}
-            <Typography.Title level={2}>Suggested Content</Typography.Title>
-            {[1, 2, 3].map(x => (
-                <div key={x}>{x}</div>
-            ))}
+                <List loading={recentsPending && recentsStatus !== 'idle'} size='small' dataSource={recents} renderItem={(item, index) => (
+                    <List.Item>
+                        <List.Item.Meta title={item.name}/>
+                        <Button type='primary' href={`/activities/${item.item_type}/${item.token}`}>Quick Add</Button>
+                    </List.Item>
+                )} />
+            <Typography.Title level={2}>Upcoming Items</Typography.Title>
+                <List 
+                    loading={upcomingPending && upcomingStatus !== 'idle'} 
+                    size='small' 
+                    dataSource={upcoming} 
+                    locale={{emptyText: 'No Data', }}
```

Comment:
why bother _and/or_ why not do it on the first list?  isnt that what it says anyway?  or does this get rid of the picture and you don't like the picture or something?

Reply:
yeah it gets rid of the image, was mostly just testing it. I was also trying to figure out styling on it. mostly changing alignment but think it would just be targeting the class it renders as.

Reply:
probably hitting it with css is what youll want, agree (have you used styled-components before?)

ok - whatever you want it to be is fine, just don't forget all 3 of them!

Reply:
Oh yeah I believe I can just put a whole component in there. probably won't bother right now until we have a more fleshed out home page design.